src/kOS/Communication/RunCommand.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+using kOS.Safe.Compilation;
+
+namespace kOS.Communication
+{
+    public class RunCommand : InterProcCommand
+    {
+        public List<Opcode> Program;
+
+        public override void Execute(SharedObjects shared)
+        {
+            if (shared.Cpu != null)
+            {
+                shared.Cpu.RunProgram(Program);
+            }
+        }
+    }
+}
```

Comment:
Ah, so it's there so we don't loose the logic, but it isn't in use.  It might be good to add a comment to that effect.

I haven't spent much time looking at this yet, but I don't think it's even exposed to the scripts right now, is that correct?  We've had a few times where people have submitted bug reports on features that we just didn't document instead of actually removing access, so I'd prefer there isn't an easy way for a user to randomly try to make it work.  I think that is already the case, but thought I should mention it in case you know otherwise.